ui-admin/src/portal/PortalEnvConfigView.tsx

```diff
@@ -58,60 +73,80 @@ const PortalEnvConfigView = ({ portalContext, portalEnv }: PortalEnvConfigViewPr
-      tooltip={user?.superuser ? 'Save' : 'You do not have permission to edit these settings'}>
-      {isLoading && <LoadingSpinner/>}
-      {!isLoading && 'Save website config'}
-    </Button>
-  </form>
+  const editableLanguages = portalEnv.environmentName === 'sandbox'
+
+  return <div>
+    <form className="bg-white">
+      <h4>Website configuration ({portalContext.portal.name})</h4>
+      <p>Configure the accessibility of the landing page shown to all visitors, and sitewide properties</p>
+      <div>
+        <label className="form-label">
+          password protected <input type="checkbox" checked={config.passwordProtected}
+            onChange={e => updateConfig('passwordProtected', e.target.checked)}/>
+        </label>
+      </div>
+      <div>
+        <label className="form-label">
+          password <input type="text" className="form-control" value={config.password ?? ''}
+            onChange={e => updateConfig('password', e.target.value)}/>
+        </label>
+      </div>
+      <div>
+        <label className="form-label">
+          accepting registration
+          <input type="checkbox" checked={config.acceptingRegistration}
+            onChange={e => updateConfig('acceptingRegistration', e.target.checked)}/>
+        </label>
+      </div>
+      <div>
+        <label className="form-label">
+          participant hostname
+          <input type="text" className="form-control" value={config.participantHostname ?? ''}
+            onChange={e => updateConfig('participantHostname', e.target.value)}/>
+        </label>
+      </div>
+      <div>
+        <label className="form-label">
+          Email source address
+          <input type="text" className="form-control" value={config.emailSourceAddress ?? ''}
+            onChange={e => updateConfig('emailSourceAddress', e.target.value)}/>
+        </label>
+      </div>
+      <div>
+        <label className="form-label">
+          Default portal language
+          <Select options={languageOptions} value={selectedLanguageOption} inputId={selectLanguageInputId}
+            isDisabled={portalEnv.supportedLanguages.length < 2} aria-label={'Select a language'}
+            onChange={e => {
+              e && updateConfig('defaultLanguage', e.value.languageCode)
+              languageOnChange(e)
+            }}/>
+        </label>
+      </div>
+      <Button onClick={save}
+        variant="primary" disabled={!user?.superuser || isLoading}
+        tooltip={user?.superuser ? 'Save' : 'You do not have permission to edit these settings'}>
+        {isLoading && <LoadingSpinner/>}
+        {!isLoading && 'Save website config'}
+      </Button>
+    </form>
+    <form className="mt-5">
+      <div>
+        <h4 className="h5">
+          Supported languages <InfoPopup content={'This list determines which languages appear in the dropdown' +
+          'of the participant view.  It is up to you to create the content for that language.'}/>
+        </h4>
```

Comment:
For some reason, clicking this tooltip refreshes the page and you can't read it :'(

Reply:
ah, another button not of type "button" problem -- good catch